arwen/context/arwen.go

```diff
@@ -7,13 +7,22 @@ import (
 	"github.com/ElrondNetwork/arwen-wasm-vm/arwen"
 	"github.com/ElrondNetwork/arwen-wasm-vm/arwen/elrondapi"
 	"github.com/ElrondNetwork/arwen-wasm-vm/arwen/ethapi"
+	"github.com/ElrondNetwork/arwen-wasm-vm/config"
+	"github.com/urfave/cli"
 	"math/big"
 	"unsafe"
 
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
 	"github.com/ElrondNetwork/go-ext-wasm/wasmer"
 )
 
+// gasCostFile defines a flag for the path to the preferences toml configuration file
+var gasCostFile = cli.StringFlag{
+	Name:  "gasCost",
+	Usage: "The gas cost configuration file to load",
+	Value: "../config/config.toml",
```

Comment:
What does `../config/config.toml` resolve to? Will it really resolve to the `config.toml` in Arwen, or will it try to open a `config.toml` relative to the Node's executable?

Reply:
changed.